test/remote-method.test.js

```diff
@@ -56,6 +56,32 @@ describe('loopback:remote-method generator', function() {
     });
   });
 
+  it('adds an entry to common/models/{name}.json with nonStatic method',
+  function(done) {
+    var methodGenerator = givenMethodGenerator();
+    helpers.mockPrompt(methodGenerator, {
+      model: 'Car',
+      methodName: 'prototype.myRemote',
+      desription: 'This is my first remote method',
+      httpPath: '',
+      acceptsArg: '',
+      returnsArg: ''
+    });
+
+    methodGenerator.run(function() {
+      var definition = common.readJsonSync('common/models/car.json');
+      var methods = definition.methods || {};
+      expect(methods).to.have.property('prototype.myRemote');
+      expect(methods['prototype.myRemote']).to.eql({
```

Comment:
This is true when the project is using LoopBack 3.0, i.e. when `common.createDummyProject` creates a LB 3.0 project. However, since 3.0 was not released yet, I think we are using LB 2.0 and thus the entry should use the old convention (name + isStatic). Am I missing something?